lib/msf/core/data_store.rb

```diff
@@ -345,7 +364,7 @@ def copy
     self.keys.each do |k|
       clone.import_option(k, self[k].kind_of?(String) ? self[k].dup : self[k], @imported[k], @imported_by[k])
     end
-    clone.aliases = self.aliases
+    clone.aliases = @aliases
```

Comment:
Since we have the `attr_accessor`, I don't think this should change.

Reply:
I understand they're different, but I'm not in agreement that only this would change. By extension, I wouldn't this mean changing all of the `@` accesses to `self.` ?